metabase/metabase
# Fixed width filters

Closes https://github.com/metabase/metabase/issues/49321

### Description

Having filters that can be different widths (based on content) is asking for inconsistencies, and it was causing some issues. This PR makes all filters the same widths.

### How to verify

1. New question > Products
2. Add filter > Title > Contains
3. Type "shoes, wallet"
4. Blur the input field, the width should not change
